platform/linux-generic/odp_schedule_if.c

```diff
@@ -121,6 +168,16 @@ void odp_schedule_order_unlock(uint32_t lock_index)
 	return sched_api->schedule_order_unlock(lock_index);
 }
 
+void odp_schedule_config_init(odp_schedule_config_t *config)
+{
+	odp_schedule_capability_t capa;
+
+	odp_schedule_capability(&capa);
+
+	memset(config, 0, sizeof(*config));
+	config->num_queues = capa.queue.max_num;
```

Comment:
Default config is implementation specific. So, implementation of this function cannot be common to all implementations.

E.g. basic queue (basic and sp schedulers) has default queue size of 4k and max queue size of 8k in config file. So, queue size should be 4k after this call. Capability shows 8k. And user may change both through config file.

queue_basic: {
	# Maximum queue size. Value must be a power of two.
	max_queue_size = 8192

	# Default queue size. Value must be a power of two.
	default_queue_size = 4096
}